CHANGES.rst

```diff
@@ -42,6 +42,8 @@ Unreleased
 * Fixed a bug that would prevent the version of the Docker image of the
   ``mkdir-heapdump`` init container to be updated when a cluster is upgraded.
 
+* Made CLUSTER_BACKUP_IMAGE config optional.
```

Comment:
Usually, we add the new changelog entries at the top of the unreleased changes 🤔  Not sure how strict this is ...

Reply:
I made the change.